resources/building-mobile-extensions/event-processing.md

```diff
@@ -8,7 +8,7 @@ An event is composed of the following information:
 * Event data, which is the structured data that is associated with this particular event.   This data is retrievable as a JSON blob.
 * Versioning information, which is used to synchronize a group of events with the associated shared data, is not directly exposed to extensions.  This information is implicitly used when getting shared states.
 
-**Tip**: Extensions use the `ACPExtensionEvent` \(iOS\) / `Event` \(Android\) class to create events and access the information the modules hold.
+**Tip**: Extensions use the `ACPExtensionEvent` \(iOS\) / `Event` \(Android\) class to create events and access the information the modules hold. The Adobe Experience Cloud Platform SDK  will  lowercase the strings  like extension name, event type and source strings, and  it is recommended  to compare the extension name, event type and event source strings using ignore case whenever compared.
```

Comment:
- What does "access the information the modules hold" mean? (It's not clear.)
- Same change as above.